2 - RTB/Verbali/Interni/23-11-13/23-11-13.typ

```diff
@@ -0,0 +1,84 @@
+    //(name: "Pippo",  role: "Proponente"),
+  ),
+  oraInizio: "15:30",
+  oraFine:  "17:30",
+  firmeEsterne: "///inserire firma esterna///"   //Questa variabile va aggiunta solo se è un documento esterno, qua i partecipanti esterni andranno a firmarsi.
+)
+
+#set text(font:"New Computer Modern", size: 1.3em)
+#set align(left)
+
+
+
+= Ordine del giorno
+  - Aggiornamento del gruppo in merito alle nuove automazioni adottate;
+  - Discussione in merito al precedente sprint;
+  - Discussione in merito alle tecnologie adottate;
+  - Ripartizione dei ruoli per il nuovo sprint.
+
+= Automazioni e nuova repo
+  Il gruppo che si è dedicato nel precedente sprint ad automatizzare la repo ha 
+  finalizzato il lavoro e ha aggiornato il gruppo riguardo alle nuove funzionalità.
+  La vecchia repo è stata abbandonata in favore di una nuova (questo perchè riorganizzare 
+  la vecchia repo avrebbe richiesto troppo tempo) con una nuova organizzazione dei branch, 
+  differente rispetto alla precedente. La differenza fondamentale risiede nel fatto che prima si aveva 
+  un branch main per le release, un branch develop comune da cui si aprivano poi i vari branch feature 
+  di lavoro; ora abbiamo un branch main che racchiude tutti i pdf dei file, un branch orfano website 
+  che contiene un piccolo sito del gruppo e un altro branch orfano src contenente tutti i sorgente 
+  dei file (.typ).  
+
+
+= Retrospettiva sullo sprint 
+  Segue quindi la discussione in merito ai miglioramenti da attuare nei seguenti sprint e quello 
+  attuale. Lo sprint passato ha avuto due criticità fondamentali: una generale confusione da 
+  parte del team di documentazione in merito alla priorità da dare ai vari documenti da redigere 
+  e cosa riportarvi all'interno; l'impossibilità di utilizzare la nuova repo bloccata dai 
+  rallentamenti subiti dal sottovalutare la mole di lavoro riguardo l'attuazione delle automazioni  
+
+= Nuove tecnologie
+  Il gruppo ha continuato la riunione facendo una panoramica sulle nuove tecnologie adottate:
+  - Jira: ITS sostitutivo a github, scelto per le molte possibilità che offre come i 
+  diagrammi di Gantt o la possibilità di definire macro obbiettivi (epic) oltre le normali 
+  task. Un altro pregio fondamentale di Jira è la possibilità di definire sprint settimanali 
+  che permettono di definire obbiettivi da raggiungere entro l'arco di, appunto, una settimana.
+  Questa feature è fondamentale per definire la qualità del lavoro svolto, se il ritmo di 
+  lavoro è stato opportunamente calcolato/rispettato, dove risiedono le criticità della 
+  pianificazione del lavoro;
+  - Miro: Una board che verrà utilizzata durante i meeting interni per supportare processi di 
+  brainstorming o per riportare concetti o criticità durante l'analisi dello sprint appena terminato.
+
+= Ripartizione dei ruoli
+  In seguito al diario di bordo della mattina abbiamo seguito i suggerimenti del professore e ripartito 
```

Comment:
In seguito al diario di bordo tenutosi in data 13-11-23, il team si è organizzato per portare avanti lo sviluppo dei tre documenti fondamentali, quali:
- analisi dei requisiti: documento di priorità maggiore. Risorse assegnate: 3 persone;
- piano di progetto. Risorse assegnate: 2 persone;
- norme di progetto: Risorse assegnate: 2 persone.